apertium-uzb.uzb.lexc

```diff
@@ -40745,3 +40800,5557 @@ Amerika% Qoʻshma% Shtatlari:Amerika% Qoʻshma% Shtatlari  NP-TOP ; ! "" ! El++
+Gazela:Gazela  NP-AL ; ! "" ! El++
+Gazprom:Gazprom  NP-AL ; ! "" ! El++
+Golos:Golos  NP-AL ; ! "" ! El++
+Grammy:Grammy  NP-AL ; ! "" ! El++
+Group:Group  NP-AL ; ! "" ! El++
+http:http  NP-AL ; ! "" ! El++
+ICAO:ICAO  NP-AL ; ! "" ! El++
+Interfaks:Interfaks  NP-AL ; ! "" ! El++
+Interneyshnl:Interneyshnl  NP-AL ; ! "" ! El++
+iOS:iOS  NP-AL ; ! "" ! El++
+Islander:Islander  NP-AL ; ! "" ! El++
+ISO:ISO  NP-AL ; ! "" ! El++
+iTunes:iTunes  NP-AL ; ! "" ! El++
+Jahon% Chempionati:Jahon% Chempionati  NP-AL ; ! "" ! El++
+Kazakhmys:Kazakhmys  NP-AL ; ! "" ! El++
+KGB:KGB  NP-AL ; ! "" ! El++
+Kirgizmambas:Kirgizmambas  NP-AL ; ! "" ! El++
+Krimenergo:Krimenergo  NP-AL ; ! "" ! El++
+Kriminform:Kriminform  NP-AL ; ! "" ! El++
+KrimSOS:KrimSOS  NP-AL ; ! "" ! El++
+Kutadgʻu% Bilig:Kutadgʻu% Bilig  NP-AL ; ! "" ! El++
+LHC:LHC  NP-AL ; ! "" ! El++
+Liga:Liga  NP-AL ; ! "" ! El++
+Linuks:Linuks  NP-AL ; ! "" ! El++
+Live:Live  NP-AL ; ! "" ! El++
+Marakana:Marakana  NP-AL ; ! "" ! El++
+Mezmurlar:Mezmurlar  NP-AL ; ! "" ! El++
+Mishkatul% Mesabih:Mishkatul% Mesabih  NP-AL ; ! "" ! El++
+MS:MS  NP-AL ; ! "" ! El++
+MSK:MSK  NP-AL ; ! "" ! El++
+MTS:MTS  NP-AL ; ! "" ! El++
+Muhakamatul% Lugʻatayn:Muhakamatul% Lugʻatayn  NP-AL ; ! "" ! El++
+Muhokamatul% Lugʻatayn:Muhokamatul% Lugʻatayn  NP-AL ; ! "" ! El++
+Nabukko:Nabukko  NP-AL ; ! "" ! El++
+Naftogaz:Naftogaz  NP-AL ; ! "" ! El++
+Nevs:Nevs  NP-AL ; ! "" ! El++
+NTV:NTV  NP-AL ; ! "" ! El++
+Ochiq% Rossiya:Ochiq% Rossiya  NP-AL ; ! "" ! El++
+OITS:OITS  NP-AL ; ! "" ! El++
+OIV:OIV  NP-AL ; ! "" ! El++
+OS:OS  NP-AL ; ! "" ! El++
+Oscar:Oscar  NP-AL ; ! "" ! El++
+OSCE:OSCE  NP-AL ; ! "" ! El++
+Otkritaya:Otkritaya  NP-AL ; ! "" ! El++
+PA:PA  NP-AL ; ! "" ! El++
+PACE:PACE  NP-AL ; ! "" ! El++
+PEN:PEN  NP-AL ; ! "" ! El++
+Pirrol:Pirrol  NP-AL ; ! "" ! El++
+Play:Play  NP-AL ; ! "" ! El++
+Premier:Premier  NP-AL ; ! "" ! El++
+PrivatBank:PrivatBank  NP-AL ; ! "" ! El++
+Qurultoy:Qurultoy  NP-AL ; ! "" ! El++
+Rav:Rav  NP-AL ; ! "" ! El++
+Records:Records  NP-AL ; ! "" ! El++
+Red:Red  NP-AL ; ! "" ! El++
+River:River  NP-AL ; ! "" ! El++
+Roskomnadzor:Roskomnadzor  NP-AL ; ! "" ! El++
+Rospotrebnadzor:Rospotrebnadzor  NP-AL ; ! "" ! El++
+Rosselhoznadzor:Rosselhoznadzor  NP-AL ; ! "" ! El++
+Rossiya% segodnya:Rossiya% segodnya  NP-AL ; ! "" ! El++
+Saxixul% Buxoriy:Saxixul% Buxoriy  NP-AL ; ! "" ! El++
+Servet%-i% Funun:Servet%-i% Funun  NP-AL ; ! "" ! El++
+Simferopol:Simferopol  NP-AL ; ! "" ! El++
+SOS:SOS  NP-AL ; ! "" ! El++
+Spravedlivaya% Rossiya:Spravedlivaya% Rossiya  NP-AL ; ! "" ! El++
+SSC:SSC  NP-AL ; ! "" ! El++
+Super% G4:Super% G4  NP-AL ; ! "" ! El++
+Svoboda:Svoboda  NP-AL ; ! "" ! El++
+Tarjimon:Tarjimon  NP-AL ; ! "" ! El++
+Tribuna:Tribuna  NP-AL ; ! "" ! El++
+TSK:TSK  NP-AL ; ! "" ! El++
+Twitter:Twitter  NP-AL ; ! "" ! El++
+Ukrenergo:Ukrenergo  NP-AL ; ! "" ! El++
+Ukrinform:Ukrinform  NP-AL ; ! "" ! El++
+Ukrtelekom:Ukrtelekom  NP-AL ; ! "" ! El++
```

Comment:
Many of these can be `<np><org>`—better than NP-AL.

Reply:
Sure, will sort them out again.

Reply:
Sorted them out, they They became NP-ALs, NP-ORGs and an NP-TOP both in tur.lexc, tur-uzb.dix and respective translations in uzb.lexc as well.